rialto_airflow/dags/harvest.py

```diff
@@ -220,50 +221,22 @@ def upload_data_quality_files(snapshot):
                 str(file_path), google_folder_id
             )
 
-    # link up dag tasks using their outputs as dependencies
-
-    snapshot = setup()
-
-    snapshot = load_authors(snapshot)
-
-    sul_pub_jsonl = sul_pub_harvest(snapshot)
+    @task_group()
+    def upload(snapshot):
+        upload_open_access_files(snapshot)
+        upload_data_quality_files(snapshot)
 
-    dimensions_jsonl = dimensions_harvest(snapshot)
+    # link up dag tasks and task groups
 
-    openalex_jsonl = openalex_harvest(snapshot)
-
-    wos_jsonl = wos_harvest(snapshot)
-
-    openalex_fill_in = fill_in_openalex(
-        snapshot, sul_pub_jsonl, openalex_jsonl, dimensions_jsonl, wos_jsonl
-    )
-
-    dimensions_fill_in = fill_in_dimensions(
-        snapshot, openalex_jsonl, dimensions_jsonl, wos_jsonl
-    )
-
-    wos_fill_in = fill_in_wos(snapshot, openalex_jsonl, dimensions_jsonl, wos_jsonl)
-
-    distilled_pubs = distill_publications(
-        snapshot, openalex_fill_in, dimensions_fill_in
-    )
-
-    linked_pubs = link_funders(
-        snapshot, openalex_fill_in, dimensions_fill_in, wos_fill_in
-    )
-
-    # link up the renaming tasks running the ones in tuples in parallel
-
-    (
-        (distilled_pubs, linked_pubs)
-        >> publish_open_access(snapshot)
-        >> upload_open_access_files(snapshot)
-    )
+    snapshot = setup()
 
```

Comment:
Using the task groups meant that the linking together of dependencies could be simplified.